src/frequenz/sdk/timeseries/_formula_engine/_formula_generators/_formula_generator.py

```diff
@@ -31,7 +31,14 @@ class ComponentNotFound(FormulaGenerationError):
 
 
 NON_EXISTING_COMPONENT_ID = sys.maxsize
-"""The component ID that does not exist in the component graph."""
+"""The component ID for non-existent components in the components graph.
+
+The non-existing component ID is commonly used in scenarios where a formula
+engine requires a component ID but there are no available components in the
+graph to associate with it. Thus, the non-existing component ID is subscribed
+instead so that the formula engine can send `None` or `0` values at the same
+frequency as the other streams.
+"""
```

Comment:
This should probably go in the commit about adding/improving attribute docs.

Reply:
Indeed, it was an oversight when rebasing :(

Reply:
Amended